pylabrobot/liquid_handling/backends/hamilton/STAR.py

```diff
@@ -43,6 +44,26 @@
   USE_USB = False
 
 
+def need_iswap_parked(method: Callable):
+  """Ensure that the iSWAP is in parked position before running command.
+
+  If the iSWAP is not parked, it get's parked before running the command.
+  """
+
+  @functools.wraps(method)
+  def wrapper(self, *args, **kwargs):
+    if self.iswap_parked:
+      result = method(self, *args, **kwargs)
+      self.iswap_parked = False
```

Comment:
Why would iswap become not parked after finishing a method that needs it parked?

Imo, `self.iswap_parked = False` should be set in the `get_plate`, `put_plate` and `move_plate_to_position` methods, which send the firmware commands to use the iSWAP.